db/seeds.rb

```diff
@@ -41,6 +44,7 @@
 i.helpers = [
   Wyeworker.find_by(name: "Pedro"),
   Wyeworker.find_by(name: "Ana"),
-  Wyeworker.find_by(name: "Facu")
+  Wyeworker.find_by(name: "Facu"),
```

Comment:
Just an idea, it might be easier to just have something like, for example,
i.helpers = Wyeworker.where(name: ["Pedro", "Ana", "Facu"]).to_a << Manager.find_by(name: "Cholee")

(There are a lot of ways to do this. Also by using Wyeworker you could just get all, this is why I mentioned in a previous comment that using Wyeworker in the way it's being used can cause confusions/issues and it could be clearer to have subtypes)

Reply:
We already had this discussion with Juan! We went with using `where`s in the next PR, the one about adding tests